README.md

```diff
@@ -37,6 +40,61 @@ The pivot used for the synchronization in the LSC connector is the email address
 
 The destination attribute for the LSC aliases connector is named `sources`.
 
+### Users Synchronization
+
+For example, it can be used to synchronize the users stored in the LDAP of an OBM instance to the James Server(s) of a TMail deployment.
+
+#### Architecture
+Given the following LDAP entry:
+
+```
+dn: uid=james-user, ou=people, dc=james,dc=org
+mail: james-user@james.org
+[...]
+
+dn: uid=james-user2, ou=people, dc=james,dc=org
+mail: james-user2@james.org
+[...]
+
+dn: uid=james-user3, ou=people, dc=james,dc=org
+mail: james-user3@james.org
+[...]
+```
+
+This will be represented as the following James users:  
+
+```bash
+$ curl -XGET http://ip:port/users
+
+[
+  {"username":"james-user2@james.org"},
+  {"username":"james-user4@james.org"}
+]
+```
+
+If LDAP entry with the `mail` attribute exists but not synchronized, the user will be created with a random generated password (24 characters).
```

Comment:
reading this, it is easy to do that by configuring LSC, and so to allow the user to choose:
- generating random password
- synchronizing existing password

I understand that it was not a goal of your task initially, so maybe you can just create an issue to mention this.

Reply:
I just created this issue. https://github.com/lsc-project/lsc-james-plugin/issues/2
Maybe, I can implement it when having a plan.